js/data/searchProviders.js

```diff
@@ -13,12 +14,19 @@ const getFaviconUrl = (name, ext = 'ico') => {
   return `chrome-extension://${braveExtensionId}/img/favicons/${name}.${ext}`
 }
 
+/**
+ * Returns path of a favicon resource
+ */
+const getPath = (name, ext = 'ico') => {
+  return path.join(__dirname, '..', '..', 'img', 'favicons', `${name}.${ext}`)
```

Comment:
verified this works on packaged builds as well as in dev

Reply:
@diracdeltas does it not work just to use the same `localImage` property which calls `getFaviconUrl`?